CHANGELOG.md

```diff
@@ -4,6 +4,54 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 7.0.0
+## BREAKING CHANGES
+
+* This release has fully deprecated `jinja` templates and will only accept `scriban` templates. 
+* The `onefuzz config` command has removed the `--authority` and `--tenant_domain` parameters. The only _required_ parameter for interactive use is the `--endpoint` parameters. The other values needed for authentication are now retrieved dynamically. 
+* The recording components used in the `coverage` task have been rewritten for improved source-level reporting. The task-level API has one breaking change: the `coverage_filter` field has been removed and replaced by the `module_allowlist` and `source_allowlist` fields. See [here](https://github.com/microsoft/onefuzz/blob/5bfcc4e242aa041d8c067471ee2e81904589a79e/src/agent/coverage/README.md#allowlists) for documentation of the new format.
+### Added
+* Service: Added  unmanaged nodes integration tests. [#2780](https://github.com/microsoft/onefuzz/pull/2780)
+* CLI: Added notification `get` command to retrieve specific notification definitions. [#2818](https://github.com/microsoft/onefuzz/pull/2818)
+* Agent: Added function allow-list to the coverage example exe. [#2830](https://github.com/microsoft/onefuzz/pull/2830)
+* Service: Added feature flag, validation when new notifications are created, and CLI support for migration to scriban. [#2816](https://github.com/microsoft/onefuzz/pull/2816), [#2834](https://github.com/microsoft/onefuzz/pull/2834), [#2839](https://github.com/microsoft/onefuzz/pull/2839)
+* Agent: Switch over to new `coverage` task. [#2741](https://github.com/microsoft/onefuzz/pull/2741)
+* Service: Added `--notification_config` support for dotnet templates. [#2842](https://github.com/microsoft/onefuzz/pull/2842)
+* Service: Report extension errors when deploying VM in a scaleset. [#2846](https://github.com/microsoft/onefuzz/pull/2846)
+* Service: Semantically validate notification configurations. [#2850](https://github.com/microsoft/onefuzz/pull/2850)
+* Agent: Accept optional `dir` of coverage test inputs. [#2853](https://github.com/microsoft/onefuzz/pull/2853)
+* Service/Agent: Added extra container to tasks. [#2847](https://github.com/microsoft/onefuzz/pull/2847)
+* Documentation: Document `coverage` crate and tool. [#2904](https://github.com/microsoft/onefuzz/pull/2904)
+* Agent: Add the ability for a task to gracefully shutdown when a task is stopped. [#2886](https://github.com/microsoft/onefuzz/pull/2886)
+
+### Changed
+* Service: Deprecated the job template feature. [#2798](https://github.com/microsoft/onefuzz/pull/2798)
+* Service: Deploy with scriban only, removing jinja. [#2809](https://github.com/microsoft/onefuzz/pull/2809)
```

Comment:
This one was disabled in #2839 so that we don't break notifications when the deployment rolls out. I think we can just include #2809 in the list of PRs for the breaking change.